builder/googlecompute/step_import_os_login_ssh_key_test.go

```diff
@@ -145,40 +106,6 @@ func TestStepImportOSLoginSSHKey_withGCEAndNoAccount(t *testing.T) {
 		t.Fatalf("bad action: %#v", action)
 	}
 
-	if step.accountEmail != fakeGCEEmail {
-		t.Fatalf("expected accountEmail to be %q but got %q", fakeGCEEmail, step.accountEmail)
-	}
-
-	pubKey, ok := state.GetOk("ssh_key_public_sha256")
-	if !ok {
-		t.Fatal("expected to see a public key")
-	}
-
-	sha256sum := sha256.Sum256(config.Comm.SSHPublicKey)
-	if pubKey != hex.EncodeToString(sha256sum[:]) {
-		t.Errorf("expected to see a matching public key, but got %q", pubKey)
-	}
-}
-
-func TestStepImportOSLoginSSHKey_withGCEAndAccount(t *testing.T) {
```

Comment:
If there is a test for it, then someone probably depends on it, do you think it could be a good idea to keep these ?

Reply:
Reinstated the test.